src/app/complex/complex-details/complex-header/complex-header.component.css

```diff
@@ -5,5 +5,18 @@
 
 h2 {
   overflow-wrap: break-word;
+  display: -webkit-box;
```

Comment:
Should we use the truncate pipe here for the name? Same as we do for the name in the list view? Or does the CSS properties work better here?

Reply:
The css version is definitively better as it doesn't require a mono font to specify a number of line, but it yielded to long dashed lines which were extending after the end of text, which was pretty ugly